deepchem/models/tensorgraph/layers.py

```diff
@@ -2614,3 +2614,166 @@ def distance_matrix(self, D):
     R = tf.reduce_sum(tf.multiply(D, D), 3)
     R = tf.sqrt(R)
     return R
+
+
+def AlphaShare(**kwargs):
+  """
+  This method should be used when constructing AlphaShare layers from Sluice Networks
+
+  References:
+  Sluice networks: Learning what to share between loosely related tasks
+  https://arxiv.org/abs/1705.08142
+  """
+  output_layers = []
+  alpha_share = AlphaShareLayer(**kwargs)
+  num_outputs = len(kwargs['in_layers'])
+  for num_layer in range(0, num_outputs):
+    output_layers.append(
+        LayerSplitter(in_layers=alpha_share, output_num=num_layer))
+  return output_layers
+
+
+class AlphaShareLayer(Layer):
+  """
+    Part of a sluice network. Adds alpha parameters to control
```

Comment:
The indentation here looks off. You might have an old version of `yapf`. Try `pip install --upgrade yapf` and rerunning.